userspace/libsinsp/filterchecks.cpp

```diff
@@ -4541,7 +4550,10 @@ uint8_t* sinsp_filter_check_tracer::extract(sinsp_evt *evt, OUT uint32_t* len, b
 				}
 			}
 
-			*len = strlen(res);
+			if(res)
```

Comment:
The fix makes sense since from the if above `res` may be not initialised, have you found any reasons why? Is it expected?